Sources/TransferMethod/ListTransferMethodViewController.swift

```diff
@@ -100,6 +94,14 @@ public final class ListTransferMethodViewController: UITableViewController {
         tableView.deselectRow(at: indexPath, animated: true)
     }
 
+    override public func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
+        return nil
```

Comment:
We could remove this implementation if the return value is `nil`

Reply:
We do need this method because this table is grouped. There will be some margin between the cell and the navigation bar if we do not have this method.